### ISSUE TITLE
[Programmatic Access – Make Code - Blocks] Incorrect role is defined for controls like: 'Basic', 'Input', ..present under left panel.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3664]
User Experience:
If the role for a particular control is incorrectly defined then it will be an issue for the end-users to identify the type of control he is navigating through and thus will get disoriented.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Open F12 Developer tool.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Basic, Input, Music, Led, more, Radio" controls and select it.
6. Navigate through all the controls appearing after selection of element.
7. Verify the issue.

Actual Result:
Incorrect role 'group' is defined for all button controls present under left panel, due to this screen reader is not announcing correct information.

Expected Result:
Correct role should be define as 'Button' control present under screen.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[WCAG 4.1.2 – Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html%7D)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG4 1 2_Incorrect role is define for buttons](https://user-images.githubusercontent.com/72871089/98905892-35473500-24e2-11eb-81be-76f7d3ac7caf.png)
